src/utils.js

```diff
@@ -131,6 +132,17 @@ export function ignorePrivateFunctions(list) {
 }
 
 
+/*
+ * Check a filename to make sure it's valid; used by scanners so we never
+ * accept new scanners that don't specify which file they're referencing.
+ */
+export function ensureFilenameExists(filename) {
+  if (typeof(filename) !== 'string' || filename.length < 1) {
```

Comment:
`typeof` is an operator - I think this works because it's being interpreted as  `typeof (filename)`.